lib/moab/file_signature.rb

```diff
@@ -65,6 +60,30 @@ def initialize(opts = {})
     # @return [String] The SHA256 checksum value of the file
     attribute :sha256, String, :on_save => Proc.new { |n| n.nil? ? "" : n.to_s }
 
+    KNOWN_ALGOS = {
+      md5: proc { Digest::MD5.new },
+      sha1: proc { Digest::SHA1.new },
+      sha256: proc { Digest::SHA2.new(256) }
+    }.freeze
```

Comment:
Thrilled to see this pulled out as a constant.   What do you think about making a separate, non-critical-path ticket for it to live in https://github.com/sul-dlss/moab-versioning/blob/master/lib/moab.rb  and also ensuring the constant is used not only here but anywhere else checksums are computed in the code?

Reply:
Yes, I would support that.  I think that can be part of the follow-up around configurability, since the configurability will effectively encapsulate the constant.

Reply:
@ndushay Already made, referencing your comment

Reply:
cool.  I made #143 and mentioned it in #142.  when the work is done, we'll deal w the dup ticket.